lib/apiservers/engine/backends/container.go

```diff
@@ -70,11 +70,15 @@ var (
 
 	portMapper portmap.PortMapper
 
+	cbpLock         sync.Mutex
+	containerByPort map[string]string // port:containerID
+
 	ctx = context.TODO()
 )
 
 func init() {
 	portMapper = portmap.NewPortMapper()
+	containerByPort = make(map[string]string)
```

Comment:
Question: Is it really an information that we would like to keep "global"? I would prefer to see data that belongs to a container inside container.